src/wp-includes/block-bindings/block-bindings.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Block Bindings API
+ *
+ * This file contains functions for managing block bindings in WordPress.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+
+/**
+ * Retrieves the singleton instance of WP_Block_Bindings.
+ *
+ * @since 6.5.0
+ *
+ * @return WP_Block_Bindings The WP_Block_Bindings instance.
+ */
+function wp_block_bindings() {
+	static $instance = null;
+	if ( is_null( $instance ) ) {
+		$instance = new WP_Block_Bindings();
+	}
+	return $instance;
+}
```

Comment:
Similar to:
https://github.com/WordPress/wordpress-develop/blob/fa441afd6e4bbe3aa6dbe9c43b590fb8e5f1c24d/src/wp-includes/script-modules.php#L21L29

The static method in the class works for me to, like this:

https://github.com/WordPress/wordpress-develop/blob/fa441afd6e4bbe3aa6dbe9c43b590fb8e5f1c24d/src/wp-includes/class-wp-block-type-registry.php#L194L200

Reply:
`wp_script_modules()` is arguably even worse because it does more than just returning an instance; the first time you call it, it adds hooks. I just opened https://github.com/WordPress/wordpress-develop/pull/5945 to address this

Reply:
I believe we should go with the static method in the class as it seems to be widely used.

Reply:
+1 for the static method. I've updated it in 940fb823ca6f9a2710cc7ad9fcc565a68d023661